scripts/release_scripts/cut_release_or_hotfix_branch.py

```diff
@@ -49,30 +49,32 @@
 from scripts import common
 
 
-def new_version_type(arg, pattern=re.compile(r'\d\.\d\.\d')):
-    """Checks that the new version name matches the expected pattern.
+def release_version_type(arg, pattern=re.compile(r'\d\.\d\.\d')):
```

Comment:
Functions should start with a verb. The previous name and the current name are not great.

Perhaps require_release_version_to_have_correct_format or similar.

Reply:
Done.